.env.example

```diff
@@ -2,3 +2,4 @@ PORT=3001
 DEV_SERVER_PORT=3000
 TOKEN_SECRET=changeMe
 DATABASE_URL=postgres://dev:dev@localhost/changeMe
+RAWG_API_KEY=4ac7afb1b60f401eb680d188cfd7eaef
```

Comment:
You should only put an example value in your example file 😂 . Same problem, it's public